frontend/src/components/Volunteer/volunteer.css

```diff
@@ -0,0 +1,95 @@
+* {
+  margin: 0;
+  padding: 0;
+}
+
+.messageInput {
+  width: 40%;
+}
+
+.jumbotron {
+  margin: auto 0;
+  justify-content: left;
+}
+
+span {
+  justify-content: center;
+}
+
+.main-content {
+  margin: auto;
+}
+
+.block {
+  display: block;
+}
+
+/*for the button* /
+ 
+  .button{
```

Comment:
lets delete all this commented out code

Reply:
it actually looks like we need this button style so lets fix this block so the styles take affect